examples/fluids/shallow-water/qfunctions/advection.h

```diff
@@ -185,13 +185,13 @@ CEED_QFUNCTION(SWExplicit_Advection)(void *ctx, CeedInt Q,
     // -- Height:
     // Evaluate the strong form using
     // div((h + H0) u) = u . grad(h) + (h + H0) div(u), with H0 constant
-    // or in index notation: (u_j h)_{,j} = u_j h_j + (h + H0) u_{j,j}
+    // or in index notation: (u_j h)_{,j} = u_j h_{,j} + (h + H0) u_{j,j}
     CeedScalar div_u = 0, u_dot_grad_h = 0;
-    for (CeedInt j=0; j<2; j++) {
+    for (CeedInt j=0; j<2; j++) { // we skip j=3 since dqdx[2] is zero
       CeedScalar dhdx_j = 0;
-      for (CeedInt k=0; k<2; k++) { // TODO: check indices! in this case dXdx is 2x3
+      for (CeedInt k=0; k<2; k++) {
         div_u += du[j][k] * dXdx[k][j]; // u_{j,j} = u_{j,K} X_{K,j}
-        dhdx_j += dh[k] * dXdx[k][j];
+        dhdx_j += dh[k] * dXdx[k][j];   // h_{,j} = h_K X_{K,j}
```

Comment:
@valeriabarra , I believe you got the indices right.
I hope this comment triggers the notifications.

Reply:
Thank you, @LeilaGhaffari ! I don't know why, but yes, today I received both notifications, one for the commit and one for this mention. Thanks a lot!